src/TrueLayer/MerchantAccounts/IMerchantAccountsApi.cs

```diff
@@ -31,6 +32,29 @@ public interface IMerchantAccountsApi
         /// <param name="userId">The user identifier.</param>
         /// <param name="cancellationToken">The cancellation token to cancel the operation.</param>
         /// <returns>An API response that includes details of a user.</returns>
-        Task<ApiResponse<GetPaymentSourcesResponse>> GetPaymentSources(string merchantAccountId, string userId, CancellationToken cancellationToken = default);
+        Task<ApiResponse<GetPaymentSourcesResponse>> GetPaymentSources(
+            string merchantAccountId,
+            string userId,
+            CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Get the transactions of a single merchant account.
+        /// </summary>
+        /// <param name="merchantAccountId">The merchant account identifier.</param>
+        /// <param name="from">The start date of the transaction.</param>
+        /// <param name="to">The end date of the transaction.</param>
+        /// <param name="getPaginatedResult">Indicated whether the result has to be paginated.</param>
+        /// <param name="cursor">Cursor used for pagination purposes, returned as next_cursor in the response payload of the inital request. Not required to access the first page of items.</param>
+        /// <param name="type">Filters transactions by payments or payouts (see <see cref="MerchantAccountTransactions.TransactionTypes"/>). If omitted, both payments and payouts are returned.</param>
+        /// <param name="cancellationToken">The cancellation token to cancel the operation.</param>
+        /// <returns></returns>
+        Task<ApiResponse<GetTransactionsResponse>> GetTransactions(
+            string merchantAccountId,
+            DateTimeOffset from,
+            DateTimeOffset to,
+            bool getPaginatedResult = true,
```

Comment:
Does it make sense to set this by default, without giving the option to opt-out initially (and without exposing anything here at the interface level)? I'd be surprised if there are new or old customers building the integration now without pagination. As such, I would personally treat that as an exception when/if needed (I don't have context on the clients that are currently interested). Mine is just a suggestion though, If we are on the same page, I'd suggest you to anyway double-check on this with Fede and MAP. 
On the java lib we had added an overload because we had the unpaginated endpoint already integrated :(

Reply:
good point, I will set it to `false` by default

Reply:
Let's move this on Slack